src/components/ConnectWallet.tsx

```diff
@@ -15,6 +15,12 @@ type ButtonProps = {
 };
 
 const temporaryHounds: HoundInfo[] = [];
+const map:any = new Map();
+
+function addToList(key:string,value:HoundInfo) {
```

Comment:
same comments about formatting, You can auto format this file by running prettier from the command line (It's in the script list of this project), or use `Shift + Alt + F` on VSCode.